aws/sdk/examples/sqs/src/bin/sqs-hello-world.rs

```diff
@@ -18,15 +19,21 @@ async fn main() -> Result<(), sqs::Error> {
             exit(1);
         }
     };
-    println!("sending a receiving on `{}`", queue_url);
+
+    println!(
+        "Sending and receiving messages on with URL: `{}`",
+        queue_url
+    );
 
     let rsp = client
         .send_message()
         .queue_url(&queue_url)
         .message_body("hello from my queue")
+        .message_group_id("MyGroup")
         .send()
         .await?;
-    println!("sent a message: {:#?}", rsp);
+
+    println!("Response from sending a message: {:#?}", rsp);
 
     let rcv_message_output = client
```

Comment:
can you delete line 41 while you're here? (and retest). That issue should be fixed. I'm not sure what's going on here, I'm pretty sure I manually tested this example. Did you get an error because message_group_id was missing?